charts/govuk-apps-conf/templates/router-configmap.yaml

```diff
@@ -110,15 +111,13 @@ data:
         }
 
         # Error pages
-        error_page 404 /404.html;
-
         charset utf-8;
 
-        location /404.html {
-          # Set Cache-Control headers on 404 pages since we overide those set by apps.
-          # So that we dont fall through to the default provided by the CDN.
-          add_header Cache-Control "public, max-age=30" always;
-          proxy_pass https://govuk-app-assets-{{ .Values.govukEnvironment }}.s3.eu-west-1.amazonaws.com/error_pages/404.html;
+        {{- range(list 400 401 403 404 405 406 410 422 429 500 502 503 504) }}
```

Comment:
```suggestion
        {{- range(list 400 401 403 404 405 406 410 422 429 500 502 503 504) }}
        error_page {{ . }} /{{ . }}.html;
        location /{{ . }}.html {
          proxy_pass https://govuk-app-assets-{{ $.Values.govukEnvironment }}.s3.eu-west-1.amazonaws.com/error_pages/{{ . }}.html;
          internal;
          proxy_set_header   Authorization "";
          proxy_set_header   Connection "";
          proxy_set_header   X-Real-IP $remote_addr;  # TODO: pass the actual end-client address
          proxy_hide_header  x-amz-id-2;
          proxy_hide_header  x-amz-meta-server-side-encryption;
          proxy_hide_header  x-amz-request-id;
          proxy_hide_header  x-amz-server-side-encryption;
          proxy_hide_header  x-amz-version-id;
          
          {{- if eq . 404 }}
          # Set Cache-Control headers on 404 pages since we overide those set by apps.
          # So that we dont fall through to the default provided by the CDN.
          add_header Cache-Control "public, max-age=30" always;
          {{- end }}
        }
        {{- end }}
```